assignment.js

```diff
@@ -32,7 +32,14 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
-
+let yearsOfService = 5
+let salary = 90000
+let netBonus = 0
+
+if (yearsOfService > 5) {
+  netBonus =salary * .05
```

Comment:
@jimernest90 for readability purposes use better spacing for your expressions:

`netBonus = salary * .05` instead of `netBonus =salary * .05`